packages/components/message-box/src/index.vue

```diff
@@ -170,6 +170,7 @@ export default defineComponent({
     buttonSize: {
       type: String as PropType<ComponentSize>,
       validator: isValidComponentSize,
+      default: 'default',
```

Comment:
required: false,

remove `default`, since we have ConfigProvider to provide default size.